tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -218,19 +215,20 @@ inline void end_measure(struct counters& timer) {
 }
 
 
-struct Vector_f {
-public:
+struct Vector_f
```

Comment:
Of course you have to remove the `public:`.
But my question is really why changing the position of the `{` to the next line?